app/javascript/controllers/apex_downloads_controller.js

```diff
@@ -24,4 +24,8 @@ export default class extends Controller {
 
     chart.render()
   }
+
+  disconnect() {
```

Comment:
i realized that the double/multiple chart rendering bug was because i wasn't cleaning up the DOM of the chart when navigating away. so when the back button is pushed, the OG chart would still be there (but unusable), but a new chart would be rendered. this method properly destroys the chart when the element is disconnected.